package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typed-rest-client",
-  "version": "1.7.3",
+  "version": "1.8.3",
```

Comment:
Nit: should be `1.8.0`. Also, please run npm install so that its reflected in the package-lock.json as well

Reply:
Done thank you!